man/fi_efa.7.md

```diff
@@ -200,6 +200,9 @@ These OFI runtime parameters apply only to the RDM endpoint.
 *FI_EFA_INTER_MIN_READ_WRITE_SIZE*
 : The mimimum message size for inter EFA write to use read write protocol. If firmware support RDMA read, and FI_EFA_USE_DEVICE_RDMA is 1, write requests whose size is larger than this value will use the read write protocol (Default 65536).
 
+*FI_EFA_USE_DEVICE_RDMA*
+: A flag to disable or enable RDMA features of the EFA device. When set to 1/true/yes/on, all RDMA features of the EFA device are used. When set to 0/false/no/off, libfabric will emulate all RDMA operations instead of offloading them to the EFA network device. If not set, RDMA feature support varies according to device ID/version. If FI_EFA_USE_DEVICE_RDMA is set to 1/true/yes/on, but the EFA device does not support RDMA, user's application is aborted and a warning message is printed. 
```

Comment:
When set to 0/false/no/off, libfabric will emulate all fi_rma operations instead of offloading them to the EFA network device. libfabric will not use device RDMA to implement send/receive